.github/workflows/main.yml

```diff
@@ -91,3 +91,25 @@ jobs:
         RESULT_URL=$(cat response.txt |grep Location |awk '{print $2}')
         sleep 2
         http --ignore-stdin $RESULT_URL
+  wasm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Should not be needed anymore when https://github.com/NLESC-JCER/cpp2wasm/issues/1 is fixed
+      - name: Generate source code
+        uses: docker://nlesc/pandoc-tangle
+        with:
+          args: README.md INSTALL.md
+
+      - name: Install emscripten
+        uses: mymindstorm/setup-emsdk@v4
+
+      - name: Build WebAssembly module
+        run: make build-wasm
+
+      - name: Start web server for hosting files in background
+        run: make host-files &
```

Comment:
why `&` at the end?

Reply:
Web server is pushed to background so later tests (cypress) can test against that running web server.